src/prompt.rs

```diff
@@ -366,7 +374,7 @@ fn show_history(history: &History) -> Result<(), anyhow::Error> {
             }
         }
     }
-    drop(childin);
+
```

Comment:
I'm not sure what's going on here and why this is needed. Clippy said that this drop call does nothing, so I removed it.